dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/restmetrics/RestMetricsTest.java

```diff
@@ -45,32 +46,36 @@
     static final String[] METHOD_STRINGS = {
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_getMessage__",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_asyncMethod_javax_ws_rs_container_AsyncResponse_",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_getMultiParamMessage_java_lang_String_",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_getMultiParamMessage_java_lang_String_java_lang_String_",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_postMessage_java_lang_String_",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_putMessage_java_lang_String_",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_deleteMessage__",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_getCheckedException_java_lang_String_",
-                                                   "restmetrics_com_ibm_ws_jaxrs_fat_restmetrics_RestMetricsResource_getUncheckedException_java_lang_String_"};
+                                            "optionsMethod",
+                                            "headMethod",
+                                            "headFallbackMethod",
+                                            "AsyncResponse",
+                                            "getMultiParamMessage_java.lang.String",
+                                            "getMultiParamMessage_java.lang.String_java.lang.String",
+                                            "postMessage_java.lang.String",
+                                            "putMessage_java.lang.String",
+                                            "deleteMessage",
+                                            "getCheckedException_java.lang.String",
+                                            "getUncheckedException_java.lang.String"};
 
     static final String URI_CONTEXT_ROOT = "http://localhost:" + Integer.getInteger("bvt.prop.HTTP_default") +
                     "/restmetrics/rest/";
 
     private static final String METRICSWAR = "restmetrics";
 
+    private static final String METRICS_URL_STRING = "/metrics/base/REST.request";
+
     private final String MAPPEDURI = getBaseTestUri(METRICSWAR, "rest", "restmetrics");
 
     private static final int BUFFER = 300; // margin or error in ms for response times.
 
     private static final int EXPECTEDRT = 200; // Expected min. response time for a single request.
 
-    private final int[] methodCounts = new int[9];
+    private final int[] methodCounts = new int[11];
```

Comment:
```suggestion
    private final int[] methodCounts = new int[METHOD_STRINGS.length];
```

This should make it easier to add/remove methods.

Reply:
agreed